indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -193,8 +206,15 @@
         insert-time (date-util/data-create-date collection)
         insert-time (index-util/date->elastic insert-time)
         coordinate-system (get-in collection [:SpatialExtent :HorizontalSpatialDomain
-                                              :Geometry :CoordinateSystem])
-        permitted-group-ids (get-coll-permitted-group-ids context provider-id collection)]
+                                                       :Geometry :CoordinateSystem])
+        permitted-group-ids (get-coll-permitted-group-ids context provider-id collection)
+        {:keys [granule-start-date granule-end-date]} (cgac/get-coll-gran-aggregates context concept-id)
+        last-3-days (t/interval (t/minus (tk/now) (t/days 3)) (tk/now))
+        granule-end-date (when-not (and granule-end-date (t/within? last-3-days granule-end-date))
+                           granule-end-date)]
+                           ;; If the granule end date is within the last 3 days we indicate that
```

Comment:
Does this comment refer to the previous line? If so move it above that line, please.